src/app/core/basic-datatypes/date/display-date/display-date.component.html

```diff
@@ -0,0 +1,14 @@
+<ng-container
+  *ngIf="!(displayAsAnonymized || isPartiallyAnonymized); else anonymized"
```

Comment:
A bit annoying, that both of these properties are necessary. Every component that is also used in templates will have to implement both which is a bit against the idea that they can be used everywhere.

Reply:
yes, agreed. I wanted to reuse date-component's template for how month-component is displaying. Unfortunately isPartiallyAnonymized doubling as an `@Input` just gets overwritten ...